migrate.sh

```diff
@@ -64,7 +64,7 @@ for pool_ksuid in $ksuid_glob; do
     pool_order=$(zq -i zng -f text "entry.id==ksuid('$pool_ksuid') | head 1 | yield join(entry.layout.keys[0], '.') + ':' + entry.layout.order" $pools_zngs)
 
     # Look for [0-9]*.zng so snap.zng is excluded
-    branch_count=$(zq -i zng -z 'yield entry.name | sort' $pool_ksuid/branches/[0-9]*.zng | zq -i zson -f text 'uniq | count()' -)
+    branch_count=$(find $pool_ksuid/branches -type f -regex "$pool_ksuid\/branches\/[0-9].*\.zng" | xargs zq -i zng - | zq -i zng -z 'yield entry.name | sort' - | zq -i zson -f text 'uniq | count()' -)
```

Comment:
I think we can simplify this in a few places.
```suggestion
    branch_count=$(find $pool_ksuid/branches -name '[0-9]*.zng' | xargs cat | zq -i zng -f text 'by entry.name | count()' -)
```

Reply:
Thanks @nwt. I re-tested with that change and indeed it works & is faster.